### PR TITLE
added Azerbaijan language toml and wiki sample

### PR DESCRIPTION
**Pull request by damirci**
How many sentences did you get at the end?
95031 lines on the output

How did you create the blacklist file?
I followed the instructions in the README, with a few changes:
I changed the word_usage.py to make it count words with Azerbaijan language Latin alphabet characters.
 ````   # Filter out symbols
    line = re.sub("[^A-z əöüğçşıİƏÖÜĞÇŞ]", " ", line)
 ````
Blocklist word frequency is 60 and lower.

Review

[samples](https://drive.google.com/drive/folders/1dzcVYhEyONPmsH6Mm4HhCG5nY1dcVz2f)

### CODE REVIEW

**Review by MichaelKohler — CHANGES_REQUESTED** [review]
Thanks for your contribution. I've left a comment regarding the abbreviations.

I'm not sure I like the samples in the repository. I don't think we'll need those once this PR is merged. Once the review process is finished, would you mind removing those? And just a quick note: if it's easier for you, there is a Google Sheet template for reviews which makes the process a bit easier than just having text files: https://docs.google.com/spreadsheets/d/1dJpysfcwmUwR4oJuw5ttGcUFYLeTbmn50Fpufz9qx-8/edit#gid=0

File: src/rules/az.toml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+broken_whitespace = ["  ", " ,", " .", " ?", " !", " ;"]
+abbreviation_patterns = [
+  "[A-ZİƏÖÜĞÇŞ]+\\.",
+  "[A-ZİƏÖÜĞÇŞ]+\\.*[A-ZİƏÖÜĞÇŞ]",
+  # for ABŞ, BDU and similar
+  "[A-ZİƏÖÜĞÇŞ]{2,}",
+  # for s., e.ə. and similar
+  "^[a-zıəöüğçş]\\.",
+  # coommon abbr
+  "və s\\.|və i.a\\.|və b\\.|və b.k\\.|və.s\\.|və.i.a\\.|və.b\\.|e.ə\\.|e .ə\\.|E.ə\\.|E. ə\\.|ö\\.|m.ə\\.|  akad\\.|prof\\.|səh\\.|küç\\.|şəh\\.|qəs\\.|məs\\.|dok\\.|dos\\.|abrev\\.|adr\\.|km\\.|sm\\.|mm\\.|resp\\.|yun\\.|lat",
```
[the remark this answers is not in the document]
Comment (damirci) [reply src/rules/az.toml]: Thanks for comments, I applied the fixes

**Comment by damirci** [comment]
I moved samples to google docs, and I fixed the az.toml file.

File: src/rules/az.toml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+needs_punctuation_end = true
+needs_letter_start = true
+needs_uppercase_start = true
+disallowed_symbols = []
+broken_whitespace = ["  ", " ,", " .", " ?", " !", " ;"]
+abbreviation_patterns = [
+  "[A-ZİƏÖÜĞÇŞ]+\\.",
+  "[A-ZİƏÖÜĞÇŞ]+\\.*[A-ZİƏÖÜĞÇŞ]",
+  # for ABŞ, BDU and similar
+  "[A-ZİƏÖÜĞÇŞ]{2,}",
```
Comment (MichaelKohler) [inline src/rules/az.toml:16]: This should already be covered by the Regex above it?

File: src/rules/az.toml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  # for s., e.ə. and similar
+  "^[a-zıəöüğçş]\\.",
+  "\\.[A-z]",
+  # coommon abbr
```
Comment (MichaelKohler) [inline src/rules/az.toml:20]: -> common

**Review by MichaelKohler — APPROVED** [review]
Thanks for the adjustments. I've left two minor comments. Could you also post the link to the Google Sheet for review?

**Comment by damirci** [comment]
https://drive.google.com/drive/folders/1dzcVYhEyONPmsH6Mm4HhCG5nY1dcVz2f

**Review by MichaelKohler — APPROVED** [review]

**Pull request merged**